csharp/src/Apache.Arrow/C/CArrowArrayImporter.cs

```diff
@@ -29,7 +29,8 @@ public static class CArrowArrayImporter
         /// </summary>
         /// <remarks>
         /// This will call the release callback once all of the buffers in the returned
-        /// IArrowArray are disposed.
+        /// IArrowArray are disposed. If freeOnRelease is set, it will also free the memory
```

Comment:
Usually comments about the parameters go in a `/// <param name="name">` comment.